api/src/pcapi/core/finance/api.py

```diff
@@ -590,6 +590,21 @@ def _get_initial_pricing_status(
     return models.PricingStatus.VALIDATED
 
 
+def force_event_repricing(
+    event: models.FinanceEvent,
+    reason: models.PricingLogReason,
+) -> None:
+    """Cancel the pricing of the event and force it to be priced again.
+
+    This is needed when the finance event is linked to a booking whose
```

Comment:
```suggestion
    This is needed when the finance event is linked to a booking which
```
car `price` n'est pas un être humain

Reply:
@Vencespass : https://en.wikipedia.org/wiki/Inanimate_whose

:popcorn: 
;)